R/rd.r

```diff
@@ -17,6 +17,20 @@ rd_file <- function(path, pkg_path = NULL) {
   }
 }
 
+#' Translate an Rd string to its HTML output
+#'
+#' @param x Rd string. Backslashes must be double-escaped ("\\\\").
+#' @param fragment logical indicating whether this represents a complete Rd file
+#' @param ... additional arguments for as_html
+#'
+#' @examples
+#' rd2html("a\n%b\nc")
+#'
+#' rd2html("a & b")
+#'
+#' rd2html(tools:::Rd_expr_doi("10.11177/0163278703255230"))
```

Comment:
Looks like this is only available in 3.3 and greater, which is why the builds are breaking. Can you either choose a different example or protect with an if statement?